CLI/test/ressources/typedefCopy/output_expected/Client.rdz.tcbl

```diff
@@ -1,6 +1,10 @@
-      *TypeCobol_Version:[[ParserVersion]]
+      *TypeCobol_Version:v0.0.0-local
```

Comment:
Error of replacing `[[ParserVersion]]` ?

Reply:
Yes, caught in my own trap 🤦‍♂️ . Fixed in 5b1bdc5e987ef71bf14cc00bb63f28b8798f0e38.